nexus-repository-cpan/src/main/java/org/sonatype/nexus/repository/cpan/orient/internal/CpanProxyRecipe.groovy

```diff
@@ -32,7 +32,7 @@ import org.sonatype.nexus.repository.http.PartialFetchHandler
 import org.sonatype.nexus.repository.httpclient.HttpClientFacet
 import org.sonatype.nexus.repository.proxy.ProxyHandler
 import org.sonatype.nexus.repository.purge.PurgeUnusedFacet
-import org.sonatype.nexus.repository.search.SearchFacet
+import org.sonatype.nexus.repository.search.ElasticSearchFacet
```

Comment:
Did you try this instead?:

`import org.sonatype.nexus.repository.content.search.SearchFacet;`

Reply:
changed

Reply:
but how should I fix the cla issue now?

Reply:
ok, added the other mail address now

Reply:
Noice! Thanks!!